src/editor/codemirror/CodeMirror.tsx

```diff
@@ -92,6 +94,7 @@ const CodeMirror = ({
             undo: undoDepth(view.state),
             redo: redoDepth(view.state),
           });
+          setSyncStatus(SyncStatus.OUT_OF_SYNC);
```

Comment:
Dubious that CM undo/redo is the right place as other file system interactions are possible (e.g. upload/delete file) that are outside the editor. Need similar info from the FS.

But good enough for a demo.